vtrace/platforms/win32.py

```diff
@@ -1219,6 +1219,9 @@ class WindowsMixin:
     """
 
     def __init__(self):
+        # Enable SeDebugPrivilege on Windows
```

Comment:
this is done here because of dependency conflicts.  when we instantiate the WindowsMixin, we're loaded enough to want this, and not accidentally be on Linux or OSX.